File: arearectangulo.c
#include <stdio.h>

int main() {
	double base, altura;
	printf("Introduce la base del rectángulo: ");
	scanf("%lf", &base);
	printf("Introduce la altura del rectángulo: ");
	scanf("%lf", &altura);
	double area = base * altura;
	printf("El área del rectángulo es: %lf\n", area);
	return 0;
}

File: AreaRectangulo.java
import java.util.Scanner;

public class AreaRectangulo {
    public static void main(String[] args) {
       Scanner scanner = new Scanner(System.in);
       System.out.print("Introduce la base del rectángulo: ");
       double base = scanner.nextDouble();
       System.out.print("Introduce la altura del rectángulo: ");
       double altura = scanner.nextDouble();
        double area = base * altura;
        System.out.println("El área del rectángulo es: " + area);
    }
}
